scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -36,148 +36,188 @@
+                      {% for choice in form.pheno_groups.choices or [] %}
+                        <option value="{{choice[0]}}" {% if institute.phenotype_groups and choice[0].split(' ,')[0] in institute.phenotype_groups  %} selected {% endif %}>{{choice[1]}}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
+                </div>
+              </fieldset>
+            </div>
+            <!-- End of phenotype groups settings -->
 
-              <div class="col-4">
-                {{ form.gene_panels.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Select gene panels that will be available for variants filtering.") }}
-                <select multiple="multiple" name="gene_panels" class="selectpicker ml-3" data-style="btn-secondary">
-                  {% for choice in form.gene_panels.choices or [] %}
-                    <option value="{{choice[0]}}" {% if institute.gene_panels and choice[0] in institute.gene_panels.keys() %} selected {% endif %}>{{choice[1]}}</option>
-                  {% endfor %}
-                </select>
-              </div>
+            <br>
 
-              <div class="col-4">
-                {{ form.gene_panels_matching.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Limit incidental findings by looking for matching managed and causatives variants only in these panels") }}
-                <select multiple="multiple" name="gene_panels_matching" class="selectpicker ml-3" data-style="btn-secondary">
-                  {% for choice in form.gene_panels_matching.choices or [] %}
-                    <option value="{{choice[0]}}" {% if institute.gene_panels_matching and choice[0] in institute.gene_panels_matching.keys() %} selected {% endif %}>{{choice[1]}}</option>
-                  {% endfor %}
-                </select>
-              </div>
+            <!-- Cohorts settings-->
+            <div class="row mt-3 d-flex align-items-center">
+              <fieldset>
+                <legend>Cohorts</legend>
+                <div class="row">
+                  <div class="col-3">
+                   {{form.cohorts.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Categories used to subdivide patients")}}
+                  </div>
+                  <div class="col-9">
+                    <select class="form-control" id="cohort_tags" name="cohorts" multiple="true" style="width:100%;">
+                      {% for cohort in institute.cohorts %}
+                        <option value="{{ cohort }}" selected>{{ cohort }}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
+                </div>
+              </fieldset>
+            </div>
+            <!-- End of cohorts settings -->
 
-            <div>
+            <br>
```

Comment:
Consider a margin, padding or such. Like, the next row already has `mt-3` - perhaps increase it if you feel it is too small?